integration/spark/app/build.gradle

```diff
@@ -27,7 +27,7 @@ ext {
     mockitoVersion = '4.11.0'
     postgresqlVersion = '42.7.1'
     testcontainersVersion = '1.19.3'
-    micrometerVersion = '1.12.4'
+    micrometerVersion = '1.13.0'
```

Comment:
This needs to be in sync with Java client - maybe just get the dependency transitively from it?

Reply:
Within the PR I got rid of `micrometer-core` dependencies from Spark integration. It's not needed as this is already present within `openlineage-java`. This setting has only effect for `testImplementation`, so this doesn't need to be in sync with Java client and our tests will fail in case of breaking changes in version.